plugins/faustwp/includes/rest/callbacks.php

```diff
@@ -278,18 +283,13 @@ function handle_rest_process_telemetry_callback( \WP_REST_Request $request ) {
 		),
 	);
 
-	/**
-	 * This code should be uncommented once we can accept/decline the
-	 * telemetry option and an appropriate clientID is generated for the site.
-	 *
-	 * @TODO
-	 */
-
-	// @codingStandardsIgnoreStart
-	// wp_remote_post($ga_telemetry_url, [
-	// 'body' => $telemetry_body,
-	// ]);
-  // @codingStandardsIgnoreEnd
+	wp_remote_post(
+		$ga_telemetry_url,
+		array(
+			'body'     => wp_json_encode( $telemetry_body ),
+			'blocking' => false,
```

Comment:
Added this so WP doesn't wait before continuing. WP says this about the blocking param:

> Whether the calling code requires the result of the request.
If set to false, the request will be sent to the remote server,
and processing returned to the calling code immediately, the caller
will know if the request succeeded or failed, but will not receive
any response from the remote server. Default true.